docs/faq/readme.md

```diff
@@ -0,0 +1,14 @@
+---
+sidebar_position: 0
+displayed_sidebar: faqSidebar
+title: Venom Blockchain FAQs
+---
+
+## Venom Blockchain FAQs
+
+Use this section of Venom Documentation as a way to quickly find answers to the most popular questions about Venom Blockchain.
+
+Sections:
+
+- [Integration](./integration.md) - you can find here answers to questions about tools and ways to integrate your dApp with Venom Blockchain.
+- [Smart Contracts Programming](./programming.md) - a collection of popular smart contracts developers question during onboarding on Venom Blockchain.
```

Comment:
- [Smart Contracts Programming](./programming.md) - a collection of popular smart contracts developers questions during onboarding on Venom Blockchain.